.gitignore

```diff
@@ -115,3 +115,5 @@ persistence.xml
 application-local.yml
 service-account.json
 application-appengine-secrets.yml
+
+gcs-key.pk12
```

Comment:
I am pretty sure that Jakob introduced this name, but I don't like it. Firstly, the de-facto standard extension for PKCS12 keystores is ".p12", not ".pk12". Also it contains the key of the service account, which is not only valid for use with GCS, so I'd like to call it `service-account.p12`.